vllm_spyre/v1/core/scheduler.py

```diff
@@ -109,6 +113,88 @@ def schedule(self) -> "SchedulerOutput":
         outputs = super().schedule()
         return outputs
 
+    def schedule_cb(self) -> "SchedulerOutput":
```

Comment:
I would propose have separate classes `StaticBatchingSpyreScheduler` and `ContinuousBatchingSpyreScheduler` and just implementing the `schedule` function differently, rather than having two functions. This could be addressed in the PR to main though, rather than this one.